docs/olm-bundle-management.md

```diff
@@ -1,1613 +1,237 @@
-   BUNDLE_TAG ?= 1.0.7
-   ```
-
-2. **Generate new bundle**:
-   ```bash
-   make bundle BUNDLE_TAG=1.0.7
-   ```
-
-3. **Review changes**:
-   ```bash
-   git diff bundle/
-   ```
-
-4. **Commit bundle changes**:
-   ```bash
-   git add bundle/ bundle.Dockerfile
-   git commit -m "chore: bump bundle version to v1.0.7"
-   ```
-
-### Version Patches
+**Bump version:**
+```bash
+# 1. Update version
+vim Makefile  # Update BUNDLE_TAG
 
-For complex version updates across multiple files, use `hack/version_patches/`:
+# 2. Generate bundle
+make bundle BUNDLE_TAG=0.2.0
 
+# 3. Review and commit
+git diff bundle/
+git add bundle/ bundle.Dockerfile
+git commit -m "chore: bump bundle version to v0.2.0"
 ```
-hack/version_patches/
-├── 1.0.5.patch
-├── 1.0.6.patch
-└── ...
-```
-
-These patches can update:
-- Image tags in the CSV
-- Version references in documentation
-- Channel information
 
-### Semantic Versioning
+**Semantic Versioning:**
+- **Major (x.0.0)**: Breaking changes
+- **Minor (0.x.0)**: New features, backward-compatible
+- **Patch (0.0.x)**: Bug fixes
 
-Follow semantic versioning:
-- **Major (x.0.0)**: Breaking changes, incompatible API updates
-- **Minor (1.x.0)**: New features, backward-compatible
-- **Patch (1.0.x)**: Bug fixes, backward-compatible
-
-### Version in Multiple Places
-
-Ensure version consistency across:
+**Ensure version consistency across:**
 1. `Makefile` (`BUNDLE_TAG`)
-2. CSV metadata name (`lightspeed-operator.v1.0.7`)
-3. CSV spec version field
+2. CSV metadata name (`lightspeed-operator.v0.2.0`)
+3. CSV spec `version` field
 4. Bundle Dockerfile labels
-5. Related catalog entries
-
----
-
-## Bundle Validation
-
-### Automatic Validation
-
-Bundle generation automatically validates:
-
-```bash
-operator-sdk bundle validate ./bundle
-```
-
-### Manual Validation
-
-Run validation explicitly:
-
-```bash
-# Basic validation
-operator-sdk bundle validate ./bundle
-
-# Validation for OpenShift
-operator-sdk bundle validate ./bundle \
-  --select-optional suite=operatorframework \
-  --select-optional name=operatorhub
-```
-
-### Common Validation Errors
-
-#### Missing required fields
-
-```
-Error: Value : (lightspeed-operator.v1.0.7) csv.Spec.minKubeVersion not specified
-```
-
-**Fix**: Add `minKubeVersion` to CSV spec:
-```yaml
-spec:
-  minKubeVersion: 1.28.0
-```
-
-#### Invalid image references
-
-```
-Error: Value : (lightspeed-operator.v1.0.7) csv.Spec.relatedImages[0].image invalid
-```
-
-**Fix**: Ensure all images use valid references (preferably digests).
-
-#### RBAC issues
-
-```
-Error: csv.Spec.install.spec.clusterPermissions[0] invalid
-```
-
-**Fix**: Verify RBAC rules are properly formatted and include all required fields.
-
-### Validation Levels
-
-- **Errors**: Must be fixed before bundle can be used
-- **Warnings**: Should be fixed but won't prevent installation
-- **Info**: Best practice suggestions
 
 ---
 
 ## Common Tasks
 
-### Task 1: Update Operator Image
+### Update Operator Image
 
```

Comment:
besides updating the related_images file manually, we have also a script to update the related images file from a snapshot in konflux.
https://docs.google.com/document/d/1KC4v4E999bhAXz39h8vmHo55g7wv23TG7aU4EAObaYY/edit?tab=t.0

Reply:
done